TimAConner/Jot
# Create PostgreSQL database and tables

# Issue
Completes Issue #2 (Create postgres database)

# Description
Complete adding postgresql and sequelize tables for the project.  The tokens table has not been added.  That will be added when I work on authentication.  Users, Notes, Note_Dates, Options, and Keywords tables have been added.  

# Testing
1. In Postgres create a Jot database `CREATE DATABASES Jot`
1. Configure sequelize/config/config.json to have the appropriate information to connect to jot
1. Run `npm run buildDb`
1.  All five tables should now be created and have a single example note and user added including the dependencies in the Options, Keywords, and Note_Dates tables.
